src/components/list/partial-styles/_has-grid-layout.scss

```diff
@@ -0,0 +1,25 @@
+:host(.has-grid-layout) {
+    --gap: var(--list-grid-gap, 0.75rem);
+
+    .mdc-deprecated-list {
+        display: grid;
+        grid-gap: var(--gap);
+        grid-template-columns: repeat(
+            auto-fill,
+            minmax(min(100%, max(calc(var(--list-grid-item-max-width, 10rem) - var(--gap)), var(--list-grid-item-min-width, 7.5rem))), 1fr)
+        );
```

Comment:
@adrianschmidt lint messes around this section, and I'll get SASS errors when I build

Reply:
I'll have a look 👍

Reply:
I've pushed a fixup ⚡